srv/GetTangentialSecurityDistance.srv

```diff
@@ -0,0 +1,5 @@
+# Service for getting the tangential security distance
+
+# Empty response
```

Comment:
Thank you very much for your review.
I modified GetOrthogonalSecurityDistance.srv, too.